IFC4x3/PropertyConstants/s/SWISS_TÜBBING_3vXlScVwfBgv5kMIoxKGRM.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SWISS_TBBING_3vXlScVwfBgv5kMIoxKGRM" Name="SWISS_TÜBBING" UniqueId="f986f726-7faa-4bab-916e-592cbb5106d6" />
```

Comment:
I would remove this as too specific. Description.

Reply:
Kept in and added description in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)